src/test/java/com/google/api/codegen/testdata/php_client_impl_library.baseline

```diff
@@ -455,12 +454,22 @@ class LibraryServiceGapicClient
      *     @type string $serviceAddress The domain name of the API remote host.
      *                                  Default 'library-example.googleapis.com'.
      *     @type mixed $port The port on which to connect to the remote host. Default 443.
+     *     @type \Grpc\Channel $channel
+     *           A `Channel` object to be used by gRPC. If not specified, a channel will be constructed.
      *     @type \Grpc\ChannelCredentials $sslCreds
-     *           A `ChannelCredentials` for use with an SSL-enabled channel.
+     *           A `ChannelCredentials` object for use with an SSL-enabled channel.
      *           Default: a credentials object returned from
      *           \Grpc\ChannelCredentials::createSsl()
+     *           NOTE: if the $channel optional argument is specified, then this option is unused.
+     *     @type bool $forceNewChannel
+     *           If true, this forces gRPC to create a new channel instead of using a persistent channel.
+     *           Defaults to false.
+     *           NOTE: if the $channel optional argument is specified, then this option is unused.
+     *     @type \Google\Auth\CredentialsLoader $credentialsLoader
+     *                              A CredentialsLoader object created using the
```

Comment:
No, good point, fixed. (This is fixed outside toolkit by the php formatting tools, but we should still get it correct here).